WordPress/Sniffs/Theme/NoFaviconSniff.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check for hardcoded favicons instead of using core implementation.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_NoFaviconSniff implements PHP_CodeSniffer_Sniff {
+
+	const REGEX_TEMPLATE = '` (?:%s)`i';
+	const REGEX_ATTR_TEMPLATE = '%1$s=[\'"](?:%2$s)[\'"]';
+
+	protected $attribute_blacklist = array(
+		'rel' => array(
+			'shortcut',
+			'icon',
+			'apple-touch-icon',
+			'apple-touch-icon-precomposed',
+		),
```

Comment:
The previous best practice was to have `'shortcut icon'`. I don't think the reverse was ever valid. https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

I wonder what the benafits of adding extra checks would be. I expect most implementations of the favicon would all or nothing.